BIDebugEngine/BIDebugEngine/RVClasses.cpp

```diff
@@ -171,10 +171,17 @@ void Serialize(const GameData& gd, JsonArchive& ar) {
         sourcedocpos x;
         x.content = gd.get_as_string();
         ar.Serialize("value", "<code>"); //Script::getScriptFromFirstLine(x, true)
-    } else {
+    } else if (strcmp(type, "float") == 0) {
```

Comment:
prefer to compare enum type instead, string is quite inefficient and we have enum type now.